src/modules/common/components/TextButton.js

```diff
@@ -15,13 +15,14 @@ export const StyledButton = styled.button`
 
 type ButtonPropsType = {
```

Comment:
I'd suggest to handle the cursor prop too if the button is disabled: `cursor: string` with style `cursor: ${props => props.disabled ? 'default' : 'pointer'};`